scripts/extract_uart_from_commit_log

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+# Copyright (c) 2019 SiFive Inc.
+
+import sys
+import os
+import re
+
+def main(argv):
+    exp = re.compile("UART TX \((.*)\):") 
+    output = ""
+
+    if len(argv) < 2:
+        # If no argument is provided, extract UART output from stdin
+        for line in sys.stdin:
+            match = exp.match(line)
+            if match:
+                output += chr(int(match.group(1), base=16))
```

Comment:
When and how one us this part? ie from stdin?

Reply:
The main way I expect people to run the script is with `./scripts/extract_uart_from_commit_log ./path/to/sim.err`

However, in the past I've gotten some good use out of preprocessing the commit log using `grep` and whatnot, and being able to pipe the commit log into the script via stdin is helpful for that.